sparkles/tests/test_review.py

```diff
@@ -309,21 +310,21 @@ def test_roll_options_dec89_9():
     exp[48000] = [' roll   P2  n_stars improvement roll_min roll_max  add_ids   drop_ids',
                   '------ ---- ------- ----------- -------- -------- --------- ---------',
                   '287.25 3.61    0.55        0.00   287.25   287.25        --        --',
-                  '281.00 7.44    6.98        9.64   276.75   285.25 608567744        --',
+                  '281.00 7.24    6.98        9.53   276.75   285.25 608567744        --',
```

Comment:
Due to the new bad region.

Reply:
When I looked at this I was thinking if we end up adding new regions we could override proseco characteristics differently for regression tests (similar to DARK40) but given where we are in the mission that seemed unlikely to be useful.

Reply:
That's an option, but it can also be illuminating to see the effects.

Reply:
Yes.  I had planned to just revert characteristics to confirm the thought that it was a second order effect based on your comments that basically 1% of the CCD was effected.

Reply:
When searching roll options then there is much more likelihood of running into a bad region.